Libraries/LibGUI/GListView.cpp

```diff
@@ -82,6 +82,16 @@ void GListView::mousedown_event(GMouseEvent& event)
     selection().clear();
 }
 
+void GListView::set_selection_index(int index)
+{
+    if (!model())
+        return;
+
+    auto index_as_model = model()->index(index, m_model_column);
```

Comment:
`index_in_model`?

Reply:
Why is this even necessary? Whoever would use this would be better off accessing `selection` directly

Reply:
Indeed, the selection should be accessed directly instead of adding this to `GListView`.